instrumentation/net/http/httptrace/otelhttptrace/clienttrace.go

```diff
@@ -287,7 +303,7 @@ func (ct *clientTracer) gotFirstResponseByte() {
 }
 
 func (ct *clientTracer) dnsStart(info httptrace.DNSStartInfo) {
-	ct.start("http.dns", "http.dns", semconv.NetHostName(info.Host))
+	ct.start("http.dns", "http.dns", attribute.Key("net.host.name").String(info.Host))
```

Comment:
Same here.

Reply:
Fixed the same as the previous instance.